Add images and Update CSS, index and JS file

dfizzkruah93 opened the pull request:
- Update index file

- Update CSS file

-Delete script.js and add a script in a folder

- Update JS file

dfizzkruah93 committed:
Add images

Update index file

delete script.js and add script in a folder

Update JS file

ShoiraTa commented on css/style.css:
```diff
@@ -454,6 +454,113 @@ textarea {
   margin: 8px auto;
 }
 
+.modal {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 99999;
+  background-color: hsl(216, 14%, 79%, 0.8);
+  backdrop-filter: blur(4px);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-y: scroll;
+}
```
- [ ] The modal works as expected, but the mobile view looks broken. Please make sure to render the mobile view of the modal as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="325" alt="Screenshot 2022-03-09 at 18 46 48" src="https://user-images.githubusercontent.com/77038610/157465150-b6234bf5-d067-45ed-8c7f-21adc066d6dc.png"> | <img width="285" alt="Screenshot 2022-03-09 at 18 46 08" src="https://user-images.githubusercontent.com/77038610/157465021-ac0d4476-d078-4737-b4c3-eed118096da6.png">

More specifically: 

- [ ] Please make sure that the content fits the modal, currently, buttons and text are going out of modal white background (if the mobile screen size is 320px). 
- [ ] Please render only 3 technologies `HTML, CSS, javascript` in mobile view.

ShoiraTa commented on css/style.css:
```diff
@@ -454,6 +454,113 @@ textarea {
+}
+
+.modal .modal-content {
+  padding: 24px;
```
- [ ] OPTIONAL: In the desktop view I recommend giving more height to the image, because now the ratio looks too narrow: 

<img width="1394" alt="Screenshot 2022-03-09 at 18 55 39" src="https://user-images.githubusercontent.com/77038610/157466931-2c8dbccd-625d-4c84-8864-c09f4794d1d1.png">

ShoiraTa reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 
----
Hi @dfizzkruah93 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dfizzkruah93 committed:
Fix broken mobile view of the modal

Fix techomlogies section

reduce height on the desktop view

Fix content going out of the modal

gemmen29 reviewed (approved):
Hi @dfizzkruah93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dfizzkruah93 commented:
@gemmen29. Thanks for the review
